sql/moz-fx-data-marketing-prod/ga_derived/www_site_downloads_v2/schema.yaml

```diff
@@ -0,0 +1,57 @@
+fields:
+- mode: NULLABLE
+  name: date
+  type: DATE
+  description: Date of the visit
```

Comment:
These days running the `bqetl query schema update` command for an ETL with no `schema.yaml` file should generate it with the field properties in the much more readable order `name:`, `type:`, `mode:`, and then I usually add `description:` at the end.

Reply:
Good to know for future reference, thank you!! I did always think this order was odd, but I had copy/pasted from elsewhere LOL